src/pudl/transform/ferc1.py

```diff
@@ -2962,6 +2967,29 @@ def merge_xbrl_metadata(
         return df
 
 
+class ElectricOperatingAndMaintenanceFerc1TableTransformer(
+    Ferc1AbstractTableTransformer
+):
+    """Transformer class for :ref:`electric_oandm_ferc1` table."""
+
+    table_id: TableIdFerc1 = TableIdFerc1.ELECTRIC_OANDM_FERC1
+    has_unique_record_ids: bool = False
+
+    @cache_df(key="dbf")
+    def process_dbf(self, raw_dbf: pd.DataFrame) -> pd.DataFrame:
+        """Process DBF but drop a bad row that is flagged by drop_duplicates."""
+        # In 2002, utility_id_ferc1_dbf 96 reported two values for
+        # administrative_and_general_operation_expense. I found the correct value by
+        # looking at the prev_yr_amt value in 2003. This removes the incorrect row
+        start_len = len(raw_dbf)
+        raw_dbf = raw_dbf[
+            ~((raw_dbf["report_year"] == 2002) & (raw_dbf["crnt_yr_amt"] == 35990321))
+        ]
+        if (dropped := start_len - len(raw_dbf)) > 1:
+            raise AssertionError(f"More rows dropped than expected: {dropped}")
+        return super().process_dbf(raw_dbf)
```

Comment:
this is a little silly given how simple this drop is, but we've been using `targeted_drop_duplicates_dbf` (or `targeted_drop_duplicates` when it is post xbrl + dbf concatenation) to do these drops. This isn't a transformer with params or anything, but rather just a convention that I think only is in use right now in the plant in service table + i just added another for the retained earnings table. I like it bc it isolates the drop, lets you document just the drop and makes it easier for someone to scan through all the ways in which we do these lil drops.

the suggestion is: pull out this dropping of the duplicate row into a class specific `targeted_drop_duplicates_dbf`. then in this `process_dbf`, run your new `targeted_drop_duplicates_dbf` then super().process_dbf(raw_dbf).

Reply:
Will I even need the class specific `process_dbf` at that point or just the new `targeted_drop_duplicates_dbf`? Seems like `super.process_dbf(raw_dbf)` is the same as what it would do without it

Reply:
oh never mind, I didn't see that `targeted_drop_duplicates` isn't defined in the parent class.